WordPress/Classes/Networking/Remote Objects/RemoteGravatarProfile.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+open class RemoteGravatarProfile {
+    var profileID = ""
+    var hash = ""
+    var requestHash = ""
+    var profileUrl = ""
+    var preferredUsername = ""
+    var thumbnailUrl = ""
+    var name = ""
+    var displayName = ""
+
+    convenience init(dict: [String: Any]) {
+        self.init()
```

Comment:
I may be missing something, but since this isn't an NSObject, we don't need to call (or have) an empty `init()`. And the only use so far uses `init(dict:)` so how about making that the only initializer? And once you've done that, all the `var` properties might as well become `let`. Cool?